DEI.md

```diff
@@ -0,0 +1,52 @@
+# Diversity, Equity, and Inclusion Project Statement
+
+<!---
+The DEI.md file was originally created in the CHAOSS project. This comment provides attribution of that work as defined under the MIT license. 
+-->
+
+<!---
+Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating your DEI.md file
+-->
+
+<img src="https://static.djangoproject.com/img/logos/django-logo-negative.svg" alt="Django" style="width:200px;"/>
+
+Scope of the DEI.md File
+
+Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) DEI metrics. 
+
+### [Project Access](https://chaoss.community/?p=4953)
+Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for all. Specific efforts in our project include:
+* Our Django Fellows span global time zones and work asynchronously with contributors across the world.
+* The Django documentation and admin has multiple translations.
+* All areas of Django are considered open to community contributions. 
+* Django Software Foundation membership, which recognises contributors to Django, is open to non-code contributors, including contributions through events, blog posts, and comments in issues and merge requests.
+* We have an Accessibility Team dedicated to improving the accessibility of Django.
+* DjangoCon's offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained.
+* DjangoCon's of recent years offer a free crèche and private facilities for breastfeeding.
```

Comment:
@tim-schilling I have added these lines here within the Project Access section. Is there anything else around DjangoCon's you want to draw attention to?

Reply:
Also drop the apostrophes: “DjangoCons”